app/views/CFPAdmin/showVotesForProposal.scala.html

```diff
@@ -169,38 +170,26 @@ <h3>@Messages("your.reviewer.rank")</h3>
                                 @Messages("no.more.talk")</a>
                         }
                         <br>
-                        @if(nextToBeReviewedSameTrack.nonEmpty && nextToBeReviewedSameFormat.nonEmpty) {
-                            @if(nextToBeReviewedSameTrack.head.track.id == proposal.track.id) {
-                                <a class="btn btn-success" accesskey="n" href="@routes.CFPAdmin.openForReview(nextToBeReviewedSameTrack.head.id)" title="Shortcut: Ctrl-Option-N"><i class="icon-forward"></i>
-                                    @Messages("gt.nextTalkTrack" , Messages(nextToBeReviewedSameTrack.head.track.label))</a>
-                            } else {
-                                <a class="btn btn-success" accesskey="n" href="@routes.CFPAdmin.openForReview(nextToBeReviewedSameTrack.head.id)" title="Shortcut: Ctrl-Option-N"><i class="icon-forward"></i>
-                                    @Messages("gt.nextTalk")</a>
-                            }
-                            @if(nextToBeReviewedSameFormat.head.talkType.id == proposal.talkType.id) {
-                                <a class="btn btn-success" accesskey="n" href="@routes.CFPAdmin.openForReview(nextToBeReviewedSameFormat.head.id)" title="Shortcut: Ctrl-Option-L"><i class="icon-forward"></i>
-                                    @Messages("gt.nextTalkType", Messages(proposal.talkType.id))</a>
-                            } else {
-                                <a class="btn btn-success" accesskey="n" href="@routes.CFPAdmin.openForReview(nextToBeReviewedSameTrack.head.id)" title="Shortcut: Ctrl-Option-L"><i class="icon-forward"></i>
-                                     @Messages("gt.nextTalk")</a>
-                            }
+                        @if(nextToBeReviewedSameTrackAndFormat.nonEmpty && nextToBeReviewedSameTrackAndFormat.head.track.id == proposal.track.id) {
+                            <a class="btn btn-success" accesskey="s" href="@routes.CFPAdmin.openForReview(nextToBeReviewedSameTrackAndFormat.head.id)" title="Shortcut: Ctrl-Option-S"><i class="icon-forward"></i>
+                                @Messages("gt.nextTalkTrackAndType" , Messages(proposal.talkType.id)), Messages(proposal.track.label))</a>
```

Comment:
There is a small bug @fcamblor 

<img width="1127" alt="Screenshot 2019-12-23 16 45 15" src="https://user-images.githubusercontent.com/174600/71367150-871e5e00-25a4-11ea-8c4b-daea1a9c2e30.png">


```suggestion
                                @Messages("gt.nextTalkTrackAndType" , Messages(proposal.talkType.id), Messages(proposal.track.label))</a>
```

and the result is ok

<img width="1096" alt="Screenshot 2019-12-23 16 51 04" src="https://user-images.githubusercontent.com/174600/71367179-969da700-25a4-11ea-9fdd-7fd2610681de.png">

Reply:
fixed through c1514478e8e877600bedf9b0d0aab2a655ca952a
thanks for the test :-)